__tests__/utils/fonts.test.js

```diff
@@ -1,4 +1,4 @@
-import {normalize, normalizeFontSize, scaleFontSize} from '../../src/utils/fonts'
+import {normalize, normalizeFontSize} from '../../src/utils/fonts'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to remove `scaleFontSize`. If this function is no longer used in the project, this change is appropriate. However, if it's still being used elsewhere, ensure that it's imported correctly in those places.
```diff
- import {normalize, normalizeFontSize, scaleFontSize} from '../../src/utils/fonts'
+ import {normalize, normalizeFontSize} from '../../src/utils/fonts'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->